R/db_desc.PrestoConnection.R

```diff
@@ -4,20 +4,29 @@
 # This source code is licensed under the BSD-style license found in the
 # LICENSE file in the root directory of this source tree.
 
-#' @include description_from_info.R PrestoConnection.R
-NULL
+.description_from_info <- function(info) {
```

Comment:
any reason you're putting this here as opposed to in the existing file?

Reply:
It's an auxiliary function that is only used in the `db_desc()` function. Having it defined in a separate file hurts readability without organizational efficiency gain.